docs/content/file-formats.md

```diff
@@ -61,6 +62,39 @@ spec:
 
 [CredentialSet]: /operator/glossary/#credentialset
 
+## ParameterSet
+
+See the glossary for more information about the [ParameterSet] resource.
+
+```yaml
+apiVersion: porter.sh/v1
+kind: ParameterSet
+metadata:
+  name: parameterset-sample
+spec:
+  schemaVersion: 1.0.1
+  namespace: operator
+  name: porter-test-me
+  parameters:
+    - name: test-secret
+      source:
+        value: test-value
+    - name: test-secret
+      source:
+        secret: test-secret
+```
+
+| Field                     | Required | Default                            | Description                                                 |
```

Comment:
I just noticed that this omits some of the fields on the parameterset (same for credentialset). Installation omits fields but we also explain that we are only describing new fields that are not present on the wrapped porter resource.

How about you include a sentence like we have for installation for parameterset and credential set so that it's more clear, or describe every field on the parameter set (e.g. name, namespace, schemaversion, etc).